cluster/src/main/scala/com/linkedin/norbert/jmx/JMX.scala

```diff
@@ -51,7 +51,7 @@ object JMX extends Logging {
 
   def unregister(mbean: ObjectInstance) = try {
     mbeanServer.unregisterMBean(mbean.getObjectName)
-    //synchronized { map.remove(mbean.getObjectName.getCanonicalName) } We treat the map as a mapping
+    synchronized { map.remove(mbean.getObjectName.getCanonicalName) } //We treat the map as a mapping
```

Comment:
I think the commenting is required. Previous change mistakenly commented out it.